packages/core/app/styles/navi-core/variables/colors.less

```diff
@@ -81,3 +81,7 @@
 @nv-color-background-primary:   @navi-white;
 @nv-color-background-secondary: @navi-gray-100;
 @nv-color-border:               @navi-gray-300;
+
+// Brand Palette
+// -----------
+@brand-100: #f5f8fe;
```

Comment:
from `denali` color spec

Reply:
Can we use a different variable name that says what the color is?
Since this is a type of blue, it should have a name like `navi-blue-{number}`. Or we could just use the blue we already have and add opacity.

Reply:
^ Can we address the comment above pls?